### PR TITLE
Enhancement: Implement UnwrapStrToLower mutator

### PR DESCRIPTION
**Pull request by localheinz**
This PR

- [x] Adds new feature `UnwrapStrToLower` mutator
- [x] Covered by tests
- [x] Doc PR: https://github.com/infection/site/pull/76

Related to #514.

💁‍♂️ Follows the example in https://github.com/infection/infection/pull/513.

### CODE REVIEW

File: src/Mutator/Util/MutatorProfile.php [hunk 1]
```diff
@@ -191,6 +191,7 @@
         Mutator\Unwrap\UnwrapArrayFlip::class,
         Mutator\Unwrap\UnwrapArrayMap::class,
         Mutator\Unwrap\UnwrapArrayReverse::class,
+        Mutator\Unwrap\UnwrapStrtolower::class,
```
Comment (borNfreee) [inline src/Mutator/Util/MutatorProfile.php]: Could you please name it `StrToLower` to be in line with https://github.com/infection/infection/pull/535?
Documentation should also be updated in this case

Comment (localheinz) [reply src/Mutator/Util/MutatorProfile.php]: @borNfreee 

Done!

**Comment by borNfreee** [comment]
could you please resolve conflict before merge @localheinz ?

**Comment by borNfreee** [comment]
this needs to be rebased and changed to use `yield` before merging. Could you please do that @localheinz?

**Comment by borNfreee** [comment]
Thank you @localheinz

**Comment by localheinz** [comment]
Thank you, @borNfreee!

**Pull request merged**